src/2025h1/build-std.md

```diff
@@ -0,0 +1,150 @@
+# build-std
+
+| Metadata |                  |
+| -------- | ---------------- |
+| Owner(s) | Rust team at Arm |
+| Teams    | [cargo]          |
+| Status   | Proposed         |
+
+*Arm's Rust team is @davidtwco, @adamgemmell, @jacobbramley, @JamieCunliffe and @Jamesbarford. This
+goal will be primarily worked on by @adamgemmell, but @davidtwco can always be contacted for
+updates.*
+
+## Summary
+
+Write an RFC for a minimum viable product (MVP) of build-std which has the potential to be
+stabilised once implemented (as opposed to the currently implemented MVP which is only suitable for
+experimentation and testing), and then implement it.
```

Comment:
Is the goal to *stabilize* in 6 months or just to write an RFC? The text seems to imply the former, but the "ownership and team asks" requests stabilization support from Cargo team.

Reply:
It's very unlikely that we'd stabilise this in six months, I expect we'd probably have the RFC completed in the first six months and perhaps have started on implementation. In the text you've commented on, I'm just trying to say that the MVP we produce should be one that *could* be stabilised, as opposed to the existing MVP of build-std, for example. I can cut some of the rows from the table at the bottom for the parts that we're unlikely to get to in this goal cycle.

Reply:
What are the plans for goals that might take 2/3 cycles but not long enough to take many years?

Reply:
Yes, that's fine, I am referring specifically to the "team asks" which included items like "stabilization decision". I don't think you are asking for that in this 6 month span.